package.json

```diff
@@ -24,11 +24,11 @@
   "dependencies": {
     "bluebird": "^3.5.1",
     "ramda": "0.23.x",
-    "request": "^2.74.0"
+    "request": "^2.74.0",
```

Comment:
👍I believe this is the correct thing for us to do, given that we are exporting the typings from the package (for typescript clients, it will matter for those that do not install devDependencies).